Cargo.toml

```diff
@@ -1,12 +1,13 @@
 [package]
-edition = "2021"
 name = "nftables"
 version = "0.5.0"
-license = "MIT OR Apache-2.0"
+authors = ["Jasper Wiegratz <wiegratz@uni-bremen.de>"]
+edition = "2021"
+rust-version = "1.65"
 description = "Safe abstraction for nftables JSON API. It can be used to create nftables rulesets in Rust and parse existing nftables rulesets from JSON."
-homepage = "https://namib.me/"
```

Comment:
Exactly. I'm planning to eventually migrate this project to the dedicated org since NAMIB is now rather defunct. We will discuss this elsewhere when the time comes.
For now there is no homepage except for this repo.